doc/design/devlink-params-configuration.md

```
@@ -0,0 +1,110 @@
+    ApplyOn string `json:"applyOn,omitempty"`
+}
+
+// DevlinkParams defines the parameters for devlink configuration
+type DevlinkParams struct {
+    Params []DevlinkParam `json:"params,omitempty"`
+}
+
+// SriovNetworkNodePolicySpec defines the desired state of SriovNetworkNodePolicy
+type SriovNetworkNodePolicySpec struct {
+    ...
+	// contains devlink params for NIC devices
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)